gradle-plugin/src/test/groovy/ch/silviowangler/gradle/restapi/RestApiPluginSpec.groovy

```diff
@@ -900,10 +907,10 @@ class RestApiPluginSpec extends Specification {
     javaFiles.isEmpty()
   }
 
-  private void assertPlantUmlFile(String filename, String testSetName) {
+  private void assertPlantUmlFile(String filename, String resourceName, String testSetName) {
```

Comment:
It seems that `filename` is a actually folder name and `resourceName` is the actual file name.